packages/app/src/modules/layoutValidation.js

```diff
@@ -120,6 +121,15 @@ const validatePieLayout = layout => {
     );
 };
 
+const validateSingleValueLayout = layout => {
+    validateAxis(layout.columns, errorLabels.defaultSeries);
+    validateAxis(layout.filters, errorLabels.pie.filter);
+    validateDimension(
+        layoutGetDimension(layout, DIMENSION_ID_PERIOD),
+        errorLabels.pie.pe
```

Comment:
Confirm that this should be "pie"

Reply:
It could have be if the error message is the same. But I added custom error messages for SV now, which are more appropriate.